vscode/package.json

```
@@ -31,7 +31,7 @@
     "download-wasm": "ts-node ./scripts/download-wasm-modules.ts",
     "release:dry-run": "pnpm run download-wasm && CODY_RELEASE_DRY_RUN=1 ts-node ./scripts/release.ts",
     "storybook": "storybook dev -p 6007 --no-open --no-version-updates --no-release-notes",
-    "test:e2e": "pnpm exec playwright install && tsc --build && pnpm run -s build:dev:desktop && playwright test",
+    "test:e2e": "tsc --build && { pnpm exec playwright install chromium & install_pw=$!; node dist/tsc/test/e2e/install-deps.js & install_vscode=$1; wait $install_pw && wait $install_vscode; } && pnpm run -s build:dev:desktop && playwright test",
```

```suggestion
    "test:e2e": "tsc --build && { pnpm exec playwright install chromium & install_pw_pid=$!; node dist/tsc/test/e2e/install-deps.js & install_vscode_pid=$1; wait $install_pw_pid && wait $install_vscode_pid; } && pnpm run -s build:dev:desktop && playwright test",
```

It took me a bit until I remembered that this is how you get process IDs, might save the next person 10secs if we add this to the suffix, wdyt?